modules/seqexec/server/src/test/scala/seqexec/server/tcs/GuideConfigDbSpec.scala

```diff
@@ -117,7 +117,7 @@ final class GuideConfigDbSpec extends FlatSpec {
       M1GuideConfig.M1GuideOn(M1Source.GAOS),
       M2GuideConfig.M2GuideOn(ComaOption.ComaOn, Set(TipTiltSource.GAOS))
     ),
-    Some(Right(GemsOn(true, false, false, true, false, true, true)))
+    Some(Right(GemsOn(true, false, false, true, false, true, true, false, false)))
```

Comment:
I hate to say it, but I think it would better to have explicit types for all these `Boolean`s.  This class takes 9 `Boolean` parameters so it would be easy to get them in the wrong order and not realize it.

Reply:
I. Hate. You.